packages/cardano-services/test/NetworkInfo/DbSyncNetworkInfoProvider/NetworkInfoBuilder.test.ts

```diff
@@ -18,47 +19,43 @@ describe('NetworkInfoBuilder', () => {
   describe('queryCirculatingSupply', () => {
     test('query circulating supply', async () => {
       const result = await builder.queryCirculatingSupply();
-      expect(BigInt(result)).toBeGreaterThan(0n);
-      expect(result).toMatchSnapshot();
+      expect(BigInt(result)).toBeGreaterThan(40_000_000_000_000_000n);
     });
   });
 
   describe('queryTotalSupply', () => {
     test('query total supply', async () => {
       const maxSupply = 45_000_000_000_000_000n;
```

Comment:
I think this is ok to keep as hardcoded, but it would be better if it's max possible value so that we don't get a negative 
total supply value for any network.

Reply:
What's the max possible value? :thinking:

Reply:
I'd guess unsigned int64?

Reply:
You can more efficiently fetch the most recent epoch's `ada_pots` row and add all the `lovelace` columns to produce the max supply.


https://github.com/input-output-hk/cardano-db-sync/blob/master/doc/schema.md#ada_pots